src/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -269,7 +269,9 @@ private function publishUpdate($object, array $options, string $type): void
             $data = json_encode(['@id' => $object->id]);
         } else {
             $resourceClass = $this->getObjectClass($object);
-            $context = $options['normalization_context'] ?? $this->resourceMetadataFactory->create($resourceClass)->getOperation()->getNormalizationContext() ?? [];
+            $context = array_replace([
+                'resource_class' => $resourceClass,
+            ], $options['normalization_context'] ?? $this->resourceMetadataFactory->create($resourceClass)->getOperation()->getNormalizationContext() ?? []);
```

Comment:
can you avoid `array_replace`? Just do something like:

```
if (!isset($context['resource_class'])) {
  $context['resource_class'] = $resourceClass;
}
```

Reply:
Done.